CODEOWNERS

```diff
@@ -1 +1 @@
-* @pleo-io/frontend-ops
\ No newline at end of file
+* @pleo-io/web-core
```

Comment:
Isn't that how the team is named? :confused-annie:

Reply:
It is, but the team doesn't have write access to this repo 🙄 it's some weird GH shennaningas